src/smefit/optimize/analytic.py

```diff
@@ -147,6 +149,31 @@ def run_sampling(self):
             @ diff_sm
         )
 
+        # Compute some metrics of the fit result
+        coeffs_name = sorted(self.coefficients.free_parameters.index)
+
+        fit_result["best_fit_point"] = dict(zip(coeffs_name, coeff_best))
+
+        self.coefficients.set_free_parameters(coeff_best)
+        self.coefficients.set_constraints()
+
+        # compute max log likelihood
+        chi2_tot = chi2.compute_chi2(
+            self.loaded_datasets,
+            self.coefficients.value,
+            self.use_quad,
+            self.use_multiplicative_prescription,
+        )
+        max_logl = float(-0.5 * chi2_tot)
+
+        fit_result["max_log_likelihood"] = max_logl
+
+        gaussian_integral = np.log(np.sqrt(np.linalg.det(2 * np.pi * coeff_covmat)))
```

Comment:
There should be a minus sign in front, no? The square root enters the denominator of the likelihood.

Reply:
This is simply the integral of a gaussian like exp(-1/2 x.(Sigma^-1).x), and then I take the log of it because we are doing the logZ

Reply:
The formula should be correct, it's the same we have in colibri and there we have tested it against the numerical result. I don't think there is anything different here.

Reply:
The exponent I agree with, but the minus sign in the normalisation in front comes from the -1/2 in the power of the prefactor in
![image](https://github.com/user-attachments/assets/130f41b7-9b42-49fd-aa9d-fb2da312c0e6), or am I missing something?

Reply:
The prefactor you see there is 1/gaussian_integral, so that it is normalised, ie 1/ sqrt(2 pi * det(Sigma))